woocommerce/woocommerce
# [Product Block Editor]: replace custom preview component with BlockPreview core

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR replaces the custom `<ContentPreview />` component by the core one: BlockPreview.





Closes https://github.com/woocommerce/woocommerce/issues/41952



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set Twenty Twenty-four like the site's theme
2. Go to the Product Editor
3. Add some content to the `description` field by using the modal editor
4. **Before** Confirm provably the preview looks different in the description field, Modal editor and frontend 

Description field | Modal Editor | Frontend 
----------|----------|----------
<img src="https://github.com/woocommerce/woocommerce/assets/77539/53f737c5-b5ad-44c1-8e15-6eebdf2916e0" width="600px" /> | ![image](https://github.com/woocommerce/woocommerce/assets/77539/8a135d47-f8eb-4dc6-a021-5decaf189e43) | ![image](https://github.com/woocommerce/woocommerce/assets/77539/7612e098-2dc3-4204-b127-fda9eecbced6)

5. **After** Confirm the preview of the Description field matches with the other ones 
<img width="696" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/8b62c39f-7f7a-4a9a-aec9-3b55c73fbafc">

6. Additionally, confirm you can resize the preview by dragging the bottom-right preview corner 


https://github.com/woocommerce/woocommerce/assets/77539/37e3a7a1-9b90-4d21-b6ba-fd7f512aadc7



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
